documentation/LOAD_TESTING.md

```diff
@@ -0,0 +1,135 @@
+# Load Testing
+
+You can run load tests to calculate the maximum throughput for your deployment. This section shows how to run tests using the FIWARE load testing tool and some reports of tests run in a demo scenario.
+
+## 1. Run your own load tests
+
+1. Clone [load-tests](https://github.com/FIWARE/load-tests) repository locally.
+2. Configure the load test on [/src/test/resources/test.conf]()
```

Comment:
This link is sending us to our repo docs. I think you want to go here, right?

https://github.com/FIWARE/load-tests/blob/master/src/test/resources/test.conf

Or maybe you are just pointing to the local directory after the cloning, which then we need to remove the link